packages/nc-gui/components/smartsheet/expanded-form/index.vue

```diff
@@ -322,15 +323,18 @@ const onDeleteRowClick = () => {
 }
 
 const onConfirmDeleteRowClick = async () => {
-  showDeleteRowModal.value = false
-  await deleteRowById(primaryKey.value)
-  message.success('Row deleted')
-  if (!props.lastRow) {
-    await onNext()
-  } else if (!props.firstRow) {
-    emits('prev')
-  } else {
-    onClose()
+  try {
```

Comment:
try block is actually for testing a block of code for errors. If you don't catch errors, basically there is no need to use. Besides, `deleteRowById` would trigger `$api` call. I think it's necessary to add a catch block here.

Reply:
in delete row by id the try catch already handled inside it

Reply:
Got it. I misread the function with the same name in different store.